datafusion/physical-plan/src/coalesce_batches.rs

```diff
@@ -216,6 +218,41 @@ impl CoalesceBatchesStream {
             match input_batch {
                 Poll::Ready(x) => match x {
                     Some(Ok(batch)) => {
+                        let new_columns: Vec<Arc<dyn Array>> = batch
```

Comment:
Nit: How about using `ArrayRef` instread of `Arc<dyn Array>` eases code readability

Reply:
done, thank you @dharanad !